mkdocs.yml

```diff
@@ -61,6 +61,7 @@ nav:
           - Overview: 'features/software-templates/index.md'
           - Configuration: 'features/software-templates/configuration.md'
           - Adding your own Templates: 'features/software-templates/adding-templates.md'
+          - Input Examples: 'features/software-templates/input-examples.md'
```

Comment:
The order differs here vs the sidebar. I think sidebar order makes the most sense, i.e. putting this below "Writing Templates"

Reply:
sorry 😅